dashboard/test/ui/features/star_labs/spritelab/eyes.feature

```diff
@@ -5,6 +5,8 @@ Feature: Sprite Lab Eyes
 Scenario: Basic Sprite Lab level
   When I open my eyes to test "sprite lab eyes"
   And I start a new Sprite Lab project
+  And I wait for 3 seconds
```

Comment:
Do we need the timed wait and the wait until?

Reply:
We may not... I think the timed wait is to wait until the p5_loading element has had time to appear. otherwise we might decide we are done waiting for p5_loading to be gone, when in reality it hasn't appeared yet. You may have best perspective to answer this since I'm not quite sure how soon p5 will show the loading element once the run button has appeared (which is what we wait for at the end of `I start a new Sprit Lab project`).

If you're not sure, perhaps I could remove the 3 second wait, and you can add it back if flakiness does not go away?

Reply:
I tested loading a spritelab project on a throttled network, and the run button and loading element appear to show at the same time. If you remove the timed wait, I will keep an eye on this test and add it back if the flakiness continues.

Reply:
Great, thanks! If flakiness does not return, it's possible the timed wait could also be removed from here: https://github.com/code-dot-org/code-dot-org/blob/4008770ce7c309df7ddb39ff5064e104d1222545/dashboard/test/ui/features/star_labs/spritelab/spritelab.feature#L7